src/index.js

```diff
@@ -127,57 +123,59 @@ class RedisContentful {
 
       // Deleting all the deleted entries from redis
       if (response.deletedEntries && response.deletedEntries.length) {
-        response.deletedEntries.forEach(async entry => {
+        for (let i = 0; i < response.deletedEntries.length; i += 1) {
```

Comment:
if `forEach` was not the issue can you please replace it back with `for` loop?

Reply:
everything else LGTM 👍🏼

Reply:
It is not causing an issue in this particular use case but can potentially skip deleting deleted entries in redis.

Reply:
I think `for...of` would be appropriate here! Check [this](https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop)!

Reply:
Okay.